addons/binding/org.openhab.binding.hs110/src/main/java/org/openhab/binding/hs110/internal/HS110.java

```diff
@@ -0,0 +1,134 @@
+package org.openhab.binding.hs110.internal;
+
+import static org.eclipse.smarthome.core.library.types.OnOffType.ON;
+import static org.openhab.binding.hs110.internal.Util.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.math.BigDecimal;
+import java.net.ConnectException;
+import java.net.InetAddress;
+import java.net.Socket;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+/**
+ * TP-Link HS100
+ *
+ * @author Insxnity
+ * @copyright Copyright (c) 2016, Insxnity Development
```

Comment:
Dear github users @agent4788 and @Insxnity,

currently i am developing an addon to control TP-Link Power plugs with help of the openHAB2 Home Automation Server. 
Please have a look at https://github.com/openhab/openhab2-addons/pull/2018 if there are any license or copyright issues with code you have released in the past.
I credited your sources in https://github.com/computerlyrik/openhab2-addon-hs110/blob/master/about.html
If you like, i can add additional license information within the openHAB guidelines.

Thank you very much for your attention
@computerlyrik